### PR TITLE
[4.3] [Guided Tours] Fancy select fields are broken and not accessible in tours

### PR DESCRIPTION
**Pull request by obuisard**
Pull Request for Issue #40211 .

### Summary of Changes

Categories and tags are fancy select fields. Although tags can be selected in tours, categories can not.
This PR fixes categories and improves the tags selection fields.
Fields can be tabbed through, categories and tags selected (to select a category or tab, use the arrow keys) or added, as intended.

### Testing Instructions

Take the article tour up to the category field.
Tab through to see that the popup and the field keep focus and you can go from one to the other.
Once it has focus, the field's dropdown should open (if you tab through or click on the field). 
A category can be entered. To access already existing categories, use the arrow keys. 
Press enter on the category and the article will change to the new category.
You will loose the 'back' button in the popup, since a refresh has been made.

Go down and test the tags the same way. This time there is no refresh when selecting tags.

Take the tour for guided tours. Check the component parameter as well.

### Actual result BEFORE applying this Pull Request

You cannot select or add a category.

### Expected result AFTER applying this Pull Request

You can select or add a category.

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by khu5h1** [comment]
I have tested this item :white_check_mark: successfully on b207e33123ed04a52e153ff4c1d8cebe45512dc5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40238">issues.joomla.org/tracker/joomla-cms/40238</a>.</sub>

File: administrator/components/com_admin/sql/updates/mysql/4.3.0-2023-03-29.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+UPDATE `#__guidedtour_steps`
+SET `target` = 'joomla-field-fancy-select .choices'
+WHERE `id` = 5;
+
```
Comment (Quy) [inline administrator/components/com_admin/sql/updates/mysql/4.3.0-2023-03-29.sql:4]: Update installation files.

Comment (obuisard) [reply administrator/components/com_admin/sql/updates/mysql/4.3.0-2023-03-29.sql:4]: OMG thanks for the catch!

**Comment by obuisard** [comment]
Khushi @khu5h1, do you mind testing again? Specifically the component field for the guided tours tour on new install.
Thanks in advance, I apologize for the mistake.

**Comment by khu5h1** [comment]
I have tested this item :white_check_mark: successfully on d8d7b7fff9af77c2c6b9561e1d90bbeb26c0e380<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40238">issues.joomla.org/tracker/joomla-cms/40238</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40238">issues.joomla.org/tracker/joomla-cms/40238</a>.</sub>

**Pull request merged**